test/protractor/login.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+describe('login page', function() {
+    var form, username;
+    
+    beforeEach(function() {
+	browser.get('/#/login');
```

Comment:
Well, I've checked it, and it doesn't like /login, there is a need to put /#/login